### DISCUSSION TITLE
A better interactivity file format ipynb support

### DISCUSSION THREAD

**Discussion by momo91377** [web-infra-dev/rspress#1143] [Ideas]
Hi, we use the mdx file format to edit the content of the document now.
Recently we found that Jupyter files have better interactivity, can the rspress support the parsing of ipynb format files in the future?

**Comment by Timeless0911** [comment]
We may not going to implement it, since just like other static site generator tools (Vitepress, Vuepress, Docusaurus, etc.) do, Rspress mainly uses markdown and mdx as its main content authoring format.

You can do it by implementing a plugin to convert ipynb format to md.

**Reply by momo91377** [reply]
Got it ,thanks a lot
